SmartPlaces.Facilities/lib/IngestionManager/src/AzureDigitalTwins/AdtGraphManager.cs

```diff
@@ -0,0 +1,382 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="AdtGraphManager.cs" company="Microsoft">
+//   Copyright (c) Microsoft Corporation.  All rights reserved.
+// </copyright>
+//-----------------------------------------------------------------------
+
+namespace IngestionManager.AzureDigitalTwins
```

Comment:
Wouldn't new files that are introduced be added as `Microsoft.SmartPlaces.Facilities.` because of common.props?